### ISSUE TITLE
Plot data from the training and test set.

### ISSUE THREAD

**Issue by andrevier** [andrevier/ECG_analysis#1]
At main.py, make 2 figures with 4 plots each. One figure with elements from the training dataset and the other, from the test dataset:

- Chosen randomly. 
- The elements are chosen by the Dataframes' label (row).
- The plots from different labels with 3000 samples.
- x axis has the samples and y axis, the Volts.
- Every plot has a title with its label.

An example are in the figure of Plot Samples section of the [tutorial](https://www.mathworks.com/help/wavelet/ug/ecg-classification-using-wavelet-features.html).
